### PR TITLE
refactor: harmonize prop/enum names

### PR DESCRIPTION
**Pull request by Lukas742**
BREAKING CHANGE: __ActionSheet:__ `a11yConfig` has been renamed to `accessibilityAttributes`.
BREAKING CHANGE: __AnalyticalTable:__ The properties and values for the `AnalyticalTableSelectionMode` enum has been changed.  `SingleSelect` is now `Single` and `MultiSelect` is now `Multiple`.
BREAKING CHANGE: __AnalyticalTable:__ `a11yConfig` has been renamed to `accessibilityAttributes`.
BREAKING CHANGE: __ObjectPage:__ `a11yConfig` has been renamed to `accessibilityAttributes`.
BREAKING CHANGE: __ObjectStatus:__ `active` has been renamed to `interactive`.

### CODE REVIEW

File: docs/MigrationGuide.mdx [hunk 1]
```diff
@@ -443,16 +444,15 @@ function MyComponent() {
...
+For a better aligned API, the `showCancelButton` prop has been replaced wih the `hideCancelButton` prop and the logic has been inverted. Also, the `a11yConfig` prop has been renamed to `accessibilityAttributes`.
 
 ```tsx
 // v1
 import { ActionSheet, Button } from '@ui5/webcomponents-react';
 
 function MyComponent() {
   return (
-    <ActionSheet showCancelButton={false}>
+    <ActionSheet showCancelButton={false} accessibilityAttributes={{ a11yConfig: { role: 'menu' } }}>
```
Comment (MarcusNotheis) [inline docs/MigrationGuide.mdx:455]:
```suggestion
    <ActionSheet showCancelButton={false} accessibilityAttributes={{ actionSheetMobileContent: { role: 'menu' } }}>
```
Copy Paste Error?

File: packages/main/src/components/ActionSheet/ActionSheet.stories.tsx [hunk 2]
```diff
@@ -24,7 +24,7 @@ const meta = {
     header: {
       control: { disable: true }
     },
-    a11yConfig: { table: { category: 'A11y props' } }
+    accessibilityAttributes: { table: { category: 'A11y props' } }
```
Comment (MarcusNotheis) [inline packages/main/src/components/ActionSheet/ActionSheet.stories.tsx:27]:
```suggestion
    accessibilityAttributes: { table: { category: 'Accessibility props' } }
```
Should we avoid the abbreviation here?

File: packages/main/src/components/ObjectPage/ObjectPage.stories.tsx [hunk 3]
```diff
@@ -44,7 +44,7 @@ const meta = {
     footer: { control: { disable: true } },
     children: { control: { disable: true } },
     placeholder: { control: { disable: true } },
-    a11yConfig: { table: { category: 'A11y props' } }
+    accessibilityAttributes: { table: { category: 'A11y props' } }
```
Comment (MarcusNotheis) [inline packages/main/src/components/ObjectPage/ObjectPage.stories.tsx:47]:
```suggestion
    accessibilityAttributes: { table: { category: 'Accessibility props' } }
```

**Review by MarcusNotheis — CHANGES_REQUESTED** [review]

File: docs/MigrationGuide.mdx (hunk 1 above)
Comment (Lukas742) [reply docs/MigrationGuide.mdx]: Good catch! I accidentally replaced `actionSheetMobileContent` with `a11yConfig` instead of `accessibilityAttributes` 😄

**Review by MarcusNotheis — APPROVED** [review]

**Pull request merged**